DEVELOPMENT.md

```diff
@@ -47,32 +47,18 @@ Development on this project uses
    (`npm test`) with a
    [good commit message](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html)
    (`git commit`)
-6. Push to the branch (`git push -u origin my-new-feature`)
-7. [Create a new Pull Request](https://github.com/nsidc/search-interface/compare)
+4. Push to the branch (`git push -u origin my-new-feature`)
```

Comment:
I don't think it's a big deal, but I think markup allow for numbering lists by just giving them all the number "1", and it will take care of the details; would that make it easier, or just cause more problems?

Reply:
You can put numbers in any order and it doesn't matter. I updated them in the hopes you wouldn't ask about the weird number order. :-)